### ISSUE TITLE
Vite/SvelteKit not detecting uniorg as ESM properly.

### ISSUE THREAD

**Issue by wildwestrom** [rasendubi/uniorg#36]
```
uniorg doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module 
(i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). 
Please contact the package author to fix.
```
So uniorg recently switch to 100% ESM, which is great!
Now I don't really know what's going on here. Either Vite is detecting things wrong, or something needs to be fixed in uniorg.

**Comment by ThomasFKJorna** [comment]
Ah, the uniorg package is the only one I didn't touch when transitioning most things to ESM, forgot to add "type: module" there!

**Comment by rasendubi** [comment]
Sorry, github somehow didn't deliver a notification to me, so I missed this issue. @ThomasFKJorna thank you for jumping in and fixing it!

I merged the PR and released as [v0.5.2](https://github.com/rasendubi/uniorg/releases/tag/v0.5.2)
